api/alembic/versions/025a81a4b7bd_limit_mat_view_to_21_days.py

```diff
@@ -0,0 +1,101 @@
+"""limit mat view to 21 days
+
+Revision ID: 025a81a4b7bd
+Revises: d392e4a5a499
+Create Date: 2024-05-24 16:06:53.822488
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '025a81a4b7bd'
+down_revision = 'd392e4a5a499'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_index(
+    op.f('ix_wsm_prediction_timestamp_station_code'),
+    'weather_station_model_predictions',
+    ['prediction_timestamp', 'station_code'],
+    unique=False
+)
```

Comment:
Seems to significantly speed up the mat view query/refresh. Might be useful with `weather_station_model_predictions` growing